config/ruby-1.9.yml

```diff
@@ -5,3 +5,13 @@ Style/Encoding:
 
 Style/HashConversion:
   Enabled: false
+
+# Percent delimiters are not supported until Ruby 2.0
+Style/PercentLiteralDelimiters:
+  Enabled: false
+
+Style/SymbolArray:
+  EnforcedStyle: brackets
+
+Style/WordArray:
+  EnforcedStyle: brackets
```

Comment:
```suggestion
```

These rules are not enabled, so please remove them here.

Reply:
It seems we miscommunicate a lot.  If you have any ideas on how I can improve my issue reports I'm all ears.  I will attempt to restate the issue, but your reason for asking me to remove them **is the reason I can't**.

Reply:
@searls Fixed!